### PR TITLE
Add the distance text to the app

### PR DESCRIPTION
**Pull request by joshprzybyszewski**
A few big things happened:

- It's now an App, not a watch face
- I changed the layout name to `MainLayout` instead of `WatchFace`
- I tweaked the display a little

This isn't nearly finished, but it gets and displays the distance in the following steps:

1. Get the location using the watch's GPS
2. Send a query to [Google's nearby search system](https://developers.google.com/places/web-service/search#PlaceSearchRequests) and parse the response
3. Send a query to [Google's distance matrix system](https://developers.google.com/maps/documentation/distance-matrix/intro#DistanceMatrixRequests) and parse the response
4. Display the parsed distance.

I know there is probably a better way of doing this that doesn't require a personal API key. Gotta look into that. We want this app to be usable by *lots* of people:)

### CODE REVIEW

File: donut-app/source/donutappView.mc [hunk 1]
```diff
@@ -43,11 +57,180 @@ class donutappView extends Ui.WatchFace {
...
         // Call the parent onUpdate function to redraw the layout
         View.onUpdate(dc);
     }
+    
+    function updateDistance(dc) {
+    	if(!posDegrees || posDegrees.size() != 2) {
+    		getPosition();
+    	} else if(!nearestPlaceId) {
+	       	requestNearestDunkin();	
+		} else if(!textToNearest || metersToNearest < 0){
```
Comment (joshprzybyszewski) [inline donut-app/source/donutappView.mc]: I am disappointed in this formatting, eclipse.

File: donut-app/source/donutappView.mc [hunk 2]
```diff
@@ -124,6 +127,8 @@ class donutappView extends Ui.WatchFace {
 			getPosition();
 			return;
 		}
+		textToNearest = "we broke it";
```
Comment (joshprzybyszewski) [inline donut-app/source/donutappView.mc]: We're going to leave this in until I figure out a better way to query google...

**Comment by andrewvogler123** [comment]
+1

**Comment by joshprzybyszewski** [comment]
Still a lot of work, but this is acceptable for adding the distance

**Pull request merged**